pkg/vadmin/start_db_vc.go

```diff
@@ -66,6 +72,9 @@ func (v *VClusterOps) genStartDBOptions(s *startdb.Parms) (vops.VStartDatabaseOp
 	opts.IsEon = vstruct.MakeNullableBool(v.VDB.IsEON())
 
 	// auth options
+	opts.Key = certs.Key
+	opts.Cert = certs.Cert
+	opts.CaCert = certs.CaCert
```

Comment:
Why we need `Key and Cert option` here? I think we use UserName and Password option